frontend/app/components/encryptable/show.js

```diff
@@ -35,8 +47,18 @@ export default class ShowComponent extends Component {
   }
 
   @action
-  showPassword() {
-    this.isPasswordVisible = true;
+  showValue(value) {
```

Comment:
yea this could be prettier as well...
Maybe something like this
```
showValue(value) {
  const visibilityMap = {
    "password": "isPasswordVisible",
    "pin": "isPinVisible",
    "token": "isTokenVisible",
    "email": "isEmailVisible",
    "customAttr": "isCustomAttrVisible",
  };

  if (visibilityMap.hasOwnProperty(value)) {
    this[visibilityMap[value]] = true;
  }
}
```
this at least gets rid of all the if else jungle.
Nevertheless it isn't much more concise, maybe a different value could be passed?